samples/deployment/TelcoChurnOnnxExport/build_dev_cheatsheet.py

```
@@ -0,0 +1,257 @@
+                "> scored = (",
+                ">   spark.table(\"telco_churn_test\")",
+                ">     .withColumn(\"pred\",",
+                ">         predict(col(\"a\"), col(\"b\"))))",
+                "> scored.show()",
+            ]),
+        ],
+    },
+]
+
+
+def draw_text_block(c, x, y, max_w, lines, font, size, leading, fg):
```

Ruff (0.15.15)

[warning] Unused function argument: `max_w`

(ARG001)